devicetypes/smartthings/zwave-switch-generic.src/zwave-switch-generic.groovy

```diff
@@ -163,23 +164,30 @@ def zwaveEvent(physicalgraph.zwave.commands.crc16encapv1.Crc16Encap cmd) {
 	}
 }
 
+def zwaveEvent(physicalgraph.zwave.commands.securityv1.SecurityMessageEncapsulation cmd) {
+	def encapsulatedCommand = cmd.encapsulatedCommand([0x20: 1, 0x25: 1])
+	if (encapsulatedCommand) {
+		zwaveEvent(encapsulatedCommand)
+	}
+}
+
 def zwaveEvent(physicalgraph.zwave.Command cmd) {
 	// Handles all Z-Wave commands we aren't interested in
 	[:]
 }
 
 def on() {
-	delayBetween([
+	cmdEncap([
 			zwave.basicV1.basicSet(value: 0xFF).format(),
 			zwave.basicV1.basicGet().format()
-	])
+	], 200)
 }
 
 def off() {
-	delayBetween([
+	cmdEncap([
 			zwave.basicV1.basicSet(value: 0x00).format(),
 			zwave.basicV1.basicGet().format()
-	])
+	], 200)
```

Comment:
Since "200" is a default value in cmdEncap, You might skip it here.

Reply:
Removed tested and working.